controllers/machinehealthcheck_controller_test.go

```diff
@@ -857,15 +857,41 @@ func TestMachineHealthCheck_Reconcile(t *testing.T) {
 		}, timeout, 100*time.Millisecond)
 	})
 
-	t.Run("when a machine is paused", func(t *testing.T) {
```

Comment:
Did you mean to remove this test case (when a machine is paused) and replace it with your new one (When remediationTemplate is set and node transitions to unhealthy, new Remediation Request should be created)?

Reply:
I had the same question but then I realized that the code was replacing the following on line 259. So I assumed it was correct.
```golang
if annotations.IsPaused(cluster, t.Machine) {
...
```
So, I guess, do we mean to remove this IsPaused check?

Reply:
Thanks Warren, I had seen that and filed it in the back of my mind, but forgot to come back to it. I have asked that here - https://github.com/kubernetes-sigs/cluster-api/pull/3606/files#r497727590.

Reply:
We earlier discussed with @vincepri to pull the support for paused machined in a different PR with related tests. But now when we postpone the merge of this PR we could keep the Pause annotation check and tests.

Reply:
I need a reminder :) do you have a link to the other conversation?

Reply:
@vincepri https://github.com/kubernetes-sigs/cluster-api/pull/3606#discussion_r488763581